pkg/packagekit/updates.less

```diff
@@ -117,26 +119,35 @@ tr.security.listing-ct-item {
 }
 
 
-/* Reformat table for narrow widths */
-@media screen and (max-width: 640px) {
+/* Hide changelog summaries in narrow widths */
+@media screen and (max-width: 80ch) {
```

Comment:
Changed to be based on character width (calculated by width of 0). This works perfectly in monospaced fonts, but 80 chacters in proportional font is actually a bit longer, usually around 1.25 to 1.33 (give or take, depending on the font). So, this is _around_ 100 average characters, which is about right, considering changelogs are often written to 72 characters wide.

Yeah, it's a little fudgy, but it's based on the default font size of the body (which happens to be the same size of the font of the changelog), instead of an arbitrary pixel width.

So, in our case, 80ch is a more accurate way to accommodate the content here.